### PR TITLE
MAINT-25904 : Fix display name of document

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: apps/portlet-explorer/src/main/webapp/javascript/eXo/ecm/MultiUpload.js [hunk 1]
```diff
@@ -1213,8 +1213,9 @@
 	}
 
 	MultiUpload.prototype.updateNotice = function(fileName){
+		fileName = decodeURI(fileName);
```
Comment (ahamdi) [inline apps/portlet-explorer/src/main/webapp/javascript/eXo/ecm/MultiUpload.js:1216]: No need for this, just decodeURIComponent twice on the filename

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: apps/portlet-explorer/src/main/webapp/javascript/eXo/ecm/MultiUpload.js (hunk 1 above)
Comment (Rdidmohamedelhadi) [reply apps/portlet-explorer/src/main/webapp/javascript/eXo/ecm/MultiUpload.js]: ok

**Comment by ahamdi** [comment]
Could you rebase it on feature/maintenance ?

**Comment by Rdidmohamedelhadi** [comment]
> Could you rebase it on feature/maintenance ?

done !

**Review by ahamdi — APPROVED** [review]

**Pull request merged**